app/models/user.rb

```diff
@@ -344,6 +344,12 @@ def captcha_required? = !github_auth? && Time.current - created_at < 2.days
 
   def flair = super&.to_sym
 
+  def mentor?
+    return true if staff?
+
+    track_mentorships.exists?
```

Comment:
I think this should actually be `user.became_mentor_at`

Reply:
(There's probably a helper method on `User` already)

Reply:
> I think this should actually be user.became_mentor_at

That does imply that you don't have to be an active mentor, is that correct?

> (There's probably a helper method on User already)

Nope

Reply:
Yeah. It might be for people who want advice before starting.

There's a `user.mentor?` you can use (it's defined via roles)